agency/src/services/aries-agent/aries-agent.class.ts

```diff
@@ -420,6 +430,32 @@ export class AriesAgent {
     }
   }
 
+  private async findSchema(
+    token: string | undefined = '',
+    schema_name: string | undefined = '',
+    schema_version: string | undefined = ''
+  ): Promise<AriesSchema> {
+    try {
+      logger.debug(`Fetching details for schema with name: ${schema_name} and version: ${schema_version}`);
+      const url = (
+        `${this.acaPyUtils.getAdminUrl()}/schemas/created` +
+        `?schema_name=${encodeURIComponent(schema_name)}&schema_version=${encodeURIComponent(schema_version)}`
```

Comment:
Nit.: could we put the request parameters in the Axios request config for consistency (see lines 521-527 as reference).